### ISSUE TITLE
NotImplementedError: Unsupported ONNX ops of type: Cast,Floor

### ISSUE THREAD

**Issue by glenn-jocher** [onnx/onnx-coreml#444]
When using coremltools to convert an ONNX model that had been exported from PyTorch 1.2.

```bash
Traceback (most recent call last):
  File "/Users/glennjocher/PycharmProjects/iD/yolov3-onnx_to_coreml.py", line 244, in <module>
    main()
  File "/Users/glennjocher/PycharmProjects/iD/yolov3-onnx_to_coreml.py", line 69, in main
    yolov3_model = convert(model_proto, image_input_names=['0'], preprocessing_args={'image_scale': 1. / 255})
  File "/Users/glennjocher/PycharmProjects/iD/onnx_coreml/converter.py", line 455, in convert
    _check_unsupported_ops(graph.nodes)
  File "/Users/glennjocher/PycharmProjects/iD/onnx_coreml/converter.py", line 143, in _check_unsupported_ops
    ','.join(unsupported_op_types)))
NotImplementedError: Unsupported ONNX ops of type: Cast,Floor
```

The offending lines seem to be complaining about the Upsample module:
```bash
  %335 : Float() = onnx::Cast[to=1](%334), scope: Darknet/Upsample # /Users/glennjocher/anaconda3/envs/yolov3/lib/python3.7/site-packages/torch/nn/functional.py:2466:0
  %336 : Float() = onnx::Floor(%335), scope: Darknet/Upsample # /Users/glennjocher/anaconda3/envs/yolov3/lib/python3.7/site-packages/torch/nn/functional.py:2466:0
```

but if I look at the upsample code there is nothing to correct:
```python
        elif mdef['type'] == 'upsample':
            modules = nn.Upsample(scale_factor=int(mdef['stride']), mode='nearest')

```

**Comment by bhushan23** [comment]
@glenn-jocher looks like you are not disabling the rank5 mapping
Cast and Floor are implemented in CoreML 3.0 and hence, you need to pass the `disable_coreml_rank5_mapping=True` to convert method
check here: https://github.com/onnx/onnx-coreml/issues/415#issuecomment-518756496

**Comment by bhushan23** [comment]
@glenn-jocher closing as support for Cast and Floor is added 
Please feel free to re-open if you need any assistance with the conversion.
